### ISSUE TITLE
Create release.sh script

### ISSUE THREAD

**Issue by kakawait** [kakawait/hugo-tranquilpeak-theme#37]

**Comment by willwade** [comment]
Not sure if this helps or hinders.. https://github.com/openassistive/OpenATFrontEnd/blob/master/.travis.yml & https://github.com/openassistive/OpenATFrontEnd/blob/master/scripts/deploy_travis.sh

(I cant take credit for that.. I'm not sure where its from but it kinda works. ). I like the idea of pushing static to a cdn.

**Comment by kakawait** [comment]
Thank for links I will check that asap because I still release manually, I have to change that

**Comment by jhabdas** [comment]
@kakawait FWIW this kind of feature may be overly prescriptive in terms of meeting the broader need unless you're turning this into a boilerplate, @kakawait. I personally use [`s3_website`](https://github.com/laurilehmijoki/s3_website) for my deployments but purposely left it out of After Dark, instead opting to create a blog post describing how to use it. https://habd.as/zero-to-http-2-aws-hugo/

**Comment by kakawait** [comment]
I think issue description is not correct, this script is mainly to automate some stupid task like bumping version on every files (actually I did it manually) and create some meta/tag.

I think deployment will be done through CI process that I should add.

**Comment by jhabdas** [comment]
Got it. In that case let me introduce you to https://github.com/semantic-release/semantic-release

**Comment by jhabdas** [comment]
And if you're looking for something a little lighter weight, there's also Standard Version. I've covered the pros and cons of each as I perceive them from real life use here:
https://github.com/speedtracker/speedtracker/issues/56#issuecomment-298863896

**Comment by kakawait** [comment]
Yeah I was thinking about because actually release version is a bip chaotic.

I think until project is on _beta_ I don't take more attention on that thing but I have to in future :)

**Comment by jhabdas** [comment]
Totally. Never too soon to start automating if it's slowing you down now. FWIW, here's an example change log I produced by Standard Version, and I love it because it's in the code and shipped with the application (unlike what Semantic Release provides):

https://github.com/comfusion/after-dark/blob/master/CHANGELOG.md
